lib/bamboo/email.ex

```diff
@@ -68,11 +68,12 @@ defmodule Bamboo.Email do
   """
 
   @type address :: String.t | {String.t, String.t}
+  @type address_list :: nil | address | [address]
```

Comment:
Can you explain that more?

I don't like `any` if it can be avoided. That doesn't allow type checking anymore.